enh(release): add Centreon Web 21.04.15 RN (#1602)
centreon/centreon-documentation: versioned_docs/version-21.04/releases/centreon-core.md

## Centreon Web

### 21.04.15

> 

Reviewer: Release date: `June 10, 2022`

> 

Reviewer: ```suggestion

Release date: `June 10, 2022`

```

#### Bug Fixes

- [API] Fixed /monitoring/host endpoint to return service state
- [API] Fixed SQL syntax when retrieving service_id field
- [Business Activity] Fixed synchronization of configuration with Remote Server
- [Install] Fixed error when installing Centreon with remote DBMS
- [Remote Server] Fixed synchronization of configuration

> - [Widget] Use ACL to get list of poller to filter display of services

Reviewer 2: ```suggestion
- [Widget] The list of pollers is now filtered according to the user's ACLs
```

> - [Widget] Use ACL to get list of poller to filter display of services

Reviewer 2: ```suggestion
- [Widget] The list of pollers is now filtered according to the user's ACLs
```
